src/RuleSet/Sets/SymfonySet.php

```diff
@@ -164,9 +164,32 @@ public function getRules(): array
             'phpdoc_no_useless_inheritdoc' => true,
             'phpdoc_order' => [
                 'order' => [
```

Comment:
Is this order mentioned it Symfony's docs somewhere?

Reply:
No, not yet. Do the implementation process is to add it to the PHP CS Fixer ruleset, then submit PR into Symfony, and only then add it to the Symfony ruleset?

Reply:
Ok, why then, this particular order?

Reply:
addressed

the order is based on common PHP community usage (like param before return), importance (like internal before type) and dependence order (like import-type before type, template before extends)